### PR TITLE
Update all missing this->_lastKey update for backend cache

### PR DESCRIPTION
**Pull request by hinablue**
Add the `this->_lastKey` update for all backend cache.

Refs: #12050

### CODE REVIEW

**Comment by sergeyklay** [comment]
Could you please update CHANGELOG.md

**Comment by nsossonko** [comment]
Why not just do let this->_lastKey = lastKey, they seem the same (would save re-concatenation, not a big deal)

**Comment by hinablue** [comment]
Update the `CHANGELOG.md` and use `$this->_lastKey = lastKey;`

File: phalcon/cache/backend/mongo.zep [hunk 1]
```diff
@@ -195,6 +195,7 @@ class Mongo extends Backend implements BackendInterface
 		} else {
 			let prefix = this->_prefix;
 			let lastkey = prefix . keyName;
+			let this->_lastKey = lastKey;
```
Comment (sergeyklay) [inline phalcon/cache/backend/mongo.zep]: `lastkey` !=  `lastKey`

Comment (sergeyklay) [inline phalcon/cache/backend/mongo.zep]: @hinablue ^

**Comment by sergeyklay** [comment]
@hinablue Also could you please rebase

File: phalcon/cache/backend/mongo.zep [hunk 2]
```diff
@@ -114,74 +114,74 @@ class Mongo extends Backend implements BackendInterface
...
+
+    /**
+     * Stores cached content into the file backend and stops the frontend
+     *
+     * @param int|string keyName
+     * @param string content
+     * @param long lifetime
+     * @param boolean stopBuffer
+     */
+    public function save(keyName = null, content = null, lifetime = null, boolean stopBuffer = true) -> boolean
```
Comment (sergeyklay) [inline phalcon/cache/backend/mongo.zep]: Could you please return back tabs instead of spaces?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # [3.0.1](https://github.com/phalcon/cphalcon/releases/tag/v3.0.1) (2016-XX-XX)
+- Fixed Phalcon\Cache\Backend\Redis::flush in order to flush cache correctly
+- Add the \_lastKey update when call `save` method for all backend cache [#12050](https://github.com/phalcon/cphalcon/issues/12050)
```
Comment (sergeyklay) [inline CHANGELOG.md]: CHANGELOG must be padded on the bottom

**Comment by sergeyklay** [comment]
Thanks. Fixed in the `3.0.x` branch.

**Pull request closed without merging**